app/services/deploy_status_checker.rb

```diff
@@ -5,6 +5,21 @@ class DeployStatusChecker
   Environment = Struct.new(:env, :statuses)
 
   DEPLOYS = [
+    Deploy.new('identity-idp', 'prod', 'https://idp.prod.login.gov'),
```

Comment:
will this be an alias to `secure.login.gov` at some point? or is that a different env?